art/step/step.py

```diff
@@ -227,6 +229,12 @@ def __call__(
         """
         self.model.set_metric_calculator(metric_calculator)
         super().__call__(previous_states, datamodule, metric_calculator)
+        #save model to file
+        ModelSaver().save(self.model, self.get_step_id(), self.name)
```

Comment:
For me it would be lovely if every run would save the model. I'd really hate the fact that my best model was overwritten for the reason that I just wanted to try step with different configuration.

Reply:
Moreover, since lightning can save models by itself I wonder if this functionality is really necessary. We shouldn't write new functionalities if they are provided